src/client/fuse_ll.cc

```diff
@@ -121,7 +121,7 @@ static int getgroups(fuse_req_t req, gid_t **sgids)
 
   gid_t *gids = new (std::nothrow) gid_t[c];
   if (!gids) {
-    return -ENOMEM;
+    return -CEPHFS_ENOMEM;
```

Comment:
This is actually returning to FUSE/kernel so we want the system errno. Which reminds me... we need a translation map to convert the `CEPHFS_E...` errno to the system errno for all returns in this driver.